### PR TITLE
Add --accounts n to dapp testnet

### PR DESCRIPTION
**Pull request by nanexcool**
Creates `n` accounts when using `dapp testnet --accounts n`

All accounts are unlocked.

Works with saving and loading.

### CODE REVIEW

File: libexec/dapp/dapp---testnet-launch [hunk 1]
```diff
@@ -81,17 +88,17 @@ echo >&2 "dapp-testnet:  Chain ID: $CHAINID"
 echo >&2 "dapp-testnet:  Database: $chaindir"
 echo >&2 "dapp-testnet:  Geth log: $chaindir/geth.log"
 
-echo "$address"     > "$chaindir/config/account"
-echo "$ETH_RPC_URL" > "$chaindir/config/rpc-url"
-echo "$port"        > "$chaindir/config/node-port"
+echo "${address[@]}" > "$chaindir/config/account"
```
Comment (mbrock) [inline libexec/dapp/dapp---testnet-launch]: I think a line per account would be nice, like with `printf "%s\n" "${address[@]}"`.

File: libexec/dapp/dapp---testnet-launch [hunk 2]
```diff
@@ -115,7 +122,9 @@ ETH_FROM=$(seth rpc eth_coinbase)
 export ETH_FROM
 export ETH_KEYSTORE=$chaindir/keystore
 export ETH_PASSWORD=/dev/null
-echo >&2 "dapp-testnet:   Account: $ETH_FROM"
+#echo >&2 "dapp-testnet:   Account: $ETH_FROM"
```
Comment (mbrock) [inline libexec/dapp/dapp---testnet-launch]: Remove this line? :)

Comment (nanexcool) [reply libexec/dapp/dapp---testnet-launch]: d'oh! will do

File: libexec/dapp/dapp---testnet-launch (hunk 1 above)
Comment (nanexcool) [reply libexec/dapp/dapp---testnet-launch]: on it!

**Comment by mbrock** [comment]
There are some ShellCheck errors! :)

**Pull request merged**